prettier/README.md

```diff
@@ -0,0 +1,17 @@
+# @bufferapp/prettier-config
+
+This is the standard Prettier config file for the projects within the @bufferapp GitHub organization.
+
+## Usage
+
+```shell script
+> npm install --save-dev @bufferapp/prettier-config
+# or
+> yarn add -D @bufferapp/prettier-config
```

Comment:
> Hey @msanroman thanks for kick starting this work.
> 
> Regarding prettier, Core is only using it _integrated with eslint_ via this [plugin](https://github.com/prettier/eslint-plugin-prettier) and [config](https://github.com/prettier/eslint-config-prettier) and not run independently for the following reasons:
> 
> * eslint has two broad categories of rules: format and code check, while prettier only cares about format (and it's quite opinionated about it)
> * when working together it's possible to have rules conflicting, so eslint has a set of configs that help you disable the formatting rules from eslint and a plugin that allows you to check and fix formatting rules via prettier-eslint integration (basically eslint runs prettier under the hook).
> 
> **If the next step in standardization is to have a common eslint setup, I'd find more value on bundling both eslint and prettier configuration in a single shared module.**
> 
> However, I may be missing some cases where we will have prettier enabled and not eslint. If that's the case then this would be fine. 😄

Yes! That's a very good one. I was prepping an ESLint PR for kicking off the discussion for frontend and backend teams and sharing `base` settings between them, but maybe we can merge these into this.

Right now we have repos that don't have Prettier enabled at all, but the goal is to have both anywhere anyway 😄

Reply:
Yeeeees! 🙌